static/app/views/performance/cache/samplePanel/samplePanel.tsx

```diff
@@ -362,6 +384,22 @@ export function CacheSamplePanel() {
             />
           </ModuleLayout.Half>
 
+          <ModuleLayout.Full>
+            <Feature features="performance-sample-panel-search">
```

Comment:
Would be slightly better to lift `<Feature>` above `ModuleLayout.Full` because an empty `ModuleLayout.Full` would still cause a gap in the layout